blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+- Fix PingSource sending duplicate events when the leading adapter fails to renew its lease [#4908](https://github.com/knative/eventing/pull/4908)
+- Fix bug preventing the namespace-scoped in-memory channel to become ready. [#4906](https://github.com/knative/eventing/pull/)
+- Fix excessive number of Replicas for eventing-webhook [#5112](https://github.com/knative/eventing/pull/5112)
+- Webhook timeout for API server calls has been increased to 10 seconds. [#5175](https://github.com/knative/eventing/pull/5175)
+
+#### 🧹 Clean up
+
+- Do not set the finalizer for pingSource for pingSource controller, just deal with it in reconcilekind [#5002](https://github.com/knative/eventing/pull/5002)
+
+### Eventing Extensions
+
+
+#### Eventing Kafka Broker v0.22
+
+#### 🚨 Breaking or Notable
+
+- Kubernetes's minimum version is 1.18. [#779](https://github.com/knative-sandbox/eventing-kafka-broker/pull/779)
+
+#### 💫 New Features & Changes
+
+- Support ordered delivery.
+  - You can choose between ordered and unordered delivery through the label kafka.eventing.knative.dev/delivery.order in your triggers.
+  - Check out the docs for more details at https://knative.dev/docs/eventing/broker/kafka-broker . [#589](https://github.com/knative-sandbox/eventing-kafka-broker/pull/)
+
+#### 🐞 Bug Fixes
+
+- Add producer interceptor io.cloudevents.kafka.PartitionKeyExtensionInterceptor to provide ordered delivery based on the partitioning extension of the CloudEvents spec. [#751](https://github.com/knative-sandbox/eventing-kafka-broker/pull/751)
+- Fix unable to deploy KafkaSink without Kafka Broker installed [#714](https://github.com/knative-sandbox/eventing-kafka-broker/pull/714)
+
+### Client v0.22
+
+kn 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+
+#### META
+The compile dependencies have been updated to Knative Serving 0.22.0 and Knative Eventing 0.22.0. There are no changes in the used API version for communicating with the backend.
+
+#### Managing custom domain mappings
+
+This release adds support for CRUD management of domain mappings. I.e. you can use `kn domain` along with its sub-commands `create`, `update`, `describe`, `list` and `delete` to fully manage DomainMapping resources for the installation of custom domains for Knative services.
+
+```
+# Create a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --ref hello
+
+# Update a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --refFlags hello
+
+# List all domain mappings
+kn domain list
+
+# Delete domain mappings 'hello.example.com'
+kn domain delete hello.example.com
+```
+See `kn domain help` for more information.
+
```

Comment:
@csantanapr can you please open a docs issue for `kn domain` to get it properly documented in the site so it's not only explained here?

@rhuss can ya'll in the CLI group please come up with a process to make sure changes to the CLI like this are being communicated properly to docs? Otherwise they just don't get documented. CC @omerbensaadon

Reply:
Sorry for that. I will update the PR template to raise awareness, so that doc relevant PR should be properly communicated.

What would be the best way to communicate ? A ping within the PR or creating an issue in the docs repo ?

Reply:
@rhuss creating an issue in the docs repo would be perfect, thank you so much 🙂

Reply:
issues https://github.com/knative/docs/issues/3496 opened @abrennan89